modules/api/src/main/java/org/eclipse/xpanse/api/controllers/ServiceDeployerApi.java

```diff
@@ -68,6 +72,9 @@
 @Secured({ROLE_ADMIN, ROLE_USER})
 public class ServiceDeployerApi {
 
+    @Value("${caffeine.cache.expire.time:60}")
```

Comment:
It is better to add time uint in the configuration item name, like 'caffeine.cache.expire.seconds'.

Reply:
Please fix this comment. 
something like this "region.azs.cache.expire.time.in.minutes"